code/datums/ai_laws.dm

```diff
@@ -213,7 +213,7 @@
 
 		add_inherent_law(line)
 	if(!inherent.len) //Failsafe to prevent lawless AIs being created.
-		log_law("AI created with empty custom laws, laws set to Asimov. Please check silicon_laws.txt.")
+		log_silicon("AI created with empty custom laws, laws set to Asimov. Please check silicon_laws.txt.")
```

Comment:
but this mentions a different file that isn't
> silicon.log

Reply:
Look a few lines above, silicon_laws.txt is a config file that says what the roundstart laws are, not the logging file I'm adding.